src/NEPSolver.jl

```diff
@@ -118,18 +118,21 @@ Executes z if displaylevel>1.
         A=v->compute_Mlincomb(nept,complex(conj(λ)),compute_Mlincomb(nep,complex(λ),complex(v)))
 
         # initialization
-        x/=norm(x); v=A(x); ρ=x⋅v; q=zeros(ComplexF64,size(nep,1));
-        k=1; err=1; tol=1e-12
+        normalize!(x); v=A(x); ρ=x⋅v; q=zeros(ComplexF64,size(nep,1));
+        k=1; err=one(eltype(x)); tol=1e-12

```

Comment:
using the type of `x` instead of 1 (which will be an `Int`) for type stability

Reply:
`one(real(eltype(x)))` otherwise I think it will not work when `eltype(x)` is complex?